test/Microsoft.ServiceFabric.Services.Wcf.Tests/WcfRemotingListenerSettingsTest.cs

```diff
@@ -0,0 +1,98 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.ServiceFabric.Services.Remoting.V2.Runtime;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Wcf.Runtime;
+using Xunit;
+
+namespace Microsoft.ServiceFabric.Services.Wcf.Remoting.V2.Wcf.Runtime
+{
+    public abstract class WcfRemotingListenerSettingsTest
+    {
+        public WcfRemotingListenerSettingsTest()
+        {
+        }
+
+        public sealed class Constructor : WcfRemotingListenerSettingsTest
+        {
+            [Fact]
+            public void ShouldInitializeRemotingExceptionDepthToDefaultValue()
+            {
+                // Act
+                var settings = new WcfRemotingListenerSettings();
```

Comment:
I would make this a readonly field in the `WcfRemotingListenerSettingsTest` class, called `sut` for System Under Test and initialize it inline or, if it needs parameters that cannot be constructed inline, in the base test class constructor. That way you don't have to repeat the same statement in every test.